src/nav/MainNavBar.js

```diff
@@ -45,7 +54,10 @@ class MainNavBar extends PureComponent<Props> {
     const leftPress = editMessage ? actions.cancelEditMessage : onPressStreams;
 
     return (
-      <View style={[styles.navBar, { backgroundColor }]}>
+      <View
+        onLayout={() => { this.reloadOrientation(); }}
+        style={[this.state.navBarStyle, { backgroundColor }]}
```

Comment:
with the help of `orientation` conditionally apply height style.
like `<View style={[styles.navBar, { backgroundColor, height: orientation === 'portrait' ? PORTAIT_HEIGHT : LANDSCAPE_HEIGHT  }]}>`